puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -15,33 +15,12 @@ pipeline {
     }
 
     stages {
-        stage('Setup Environment') {
+        stage('Copy Source') {
             steps {
                 script {
-                    if ( env.git_ref == null || env.git_ref == '' ) {
-                        error("git_ref parameter is blank")
-                    }
-                }
-                setup_nightly_build_environment(
-                    git_url: git_url,
-                    git_ref: env.git_ref ?: git_ref,
-                    project_name: project_name
-                )
-            }
-        }
-        stage('Archive Git Commit') {
-            steps {
-                script {
-                    dir(project_name) {
-                        commit_hash = archive_git_hash()
-                    }
-                }
-            }
-        }
-        stage('Build and Archive Source') {
-            steps {
-                script {
-                    sourcefile_paths = generate_sourcefiles(project_name: project_name, source_type: source_type)
+                    sourcefile_paths = pwd(tmp: true)
+                    source_project_name = "${project_name}-${git_ref}-source-release"
```

Comment:
I think `${git_ref}` should be `develop` with the change to copying the files from known jobs.

Reply:
Not every job has `develop`, like katello which uses `master`. In the vars file this is set.